Sync/Synchronizers/Bookmarks/BookmarksSynchronizer.swift

```diff
@@ -89,6 +89,51 @@ open class BufferingBookmarksSynchronizer: TimestampedSingleCollectionSynchroniz
         return BookmarksStorageVersion
     }
 
+    fileprivate func createMobileRootRecord(_ storage: LocalItemSource, _ buffer: BufferItemSource, newBookmarks: [BookmarkMirrorItem]) -> Deferred<Maybe<Record<BookmarkBasePayload>>> {
```

Comment:
Perhaps name this one `buildMobileRootRecord(local: buffer: additionalChildren)`. `create` implies storage side effects.